### ISSUE TITLE
sidecar correctly does config on first mongo cluster only

### ISSUE THREAD

**Issue by aaronnewsome** [cvallance/mongo-k8s-sidecar#111]
Hello everyone. I'm a bit new to kubernetes and I thought of using this to easily configure a mongodb cluster.

Using the stateful set example here:

https://github.com/cvallance/mongo-k8s-sidecar/blob/master/example/StatefulSet/mongo-statefulset.yaml

I was able to get it up and running. However, the sidecar only configures the 1st mongodb cluster correctly. If I bring up a 2nd or 3rd cluster, the sidecar does not configure them or init the cluster. On a 2nd cluster, I can open a shell, run mongo and also run rs.initiate to initialize the cluster.

I've already made a script that I can run manually that does the rs.initiate but I'd like to figure out why the sidecar won't configure 2nd and 3rd clusters. Any ideas?

Thanks in advance.

**Comment by ratnakarreddyg** [comment]
Hello @aaronnewsome 

Could you please share a script that can perform rs.initiate? I am also having a similar requirement.

**Comment by aaronnewsome** [comment]
Yes I can share. I was teaching kubernetes to all the guys on my team. Everyone got their own copy of a 3 node mongoDB cluster. My script initialized all of the perfectly. I’ll find it and send it over.
